swift/ql/test/library-tests/controlflow/graph/Cfg.expected

```diff
@@ -596,9 +597,6 @@ cfg.swift:
 #-----|  -> exit callClosures()
 
 #   65| var ... = ...
-#-----|  -> x1
-
-#   65| x1
```

Comment:
Are we fine with the general lack of VarDecl nodes in the CFG of a `BraceStmt`? Even though they are apparently unnecessary for data flow.

Reply:
It's probably a bit unexpected from a user point-of-view, but we expect that very few people write queries directly on the control-flow graph. So it's probably fine.